src/ucp/wireup/select.c

```diff
@@ -570,13 +570,18 @@ static inline double ucp_wireup_tl_iface_latency(ucp_context_h context,
                                                  const uct_iface_attr_t *iface_attr,
                                                  const ucp_address_iface_attr_t *remote_iface_attr)
 {
+    uint8_t packed_lat;
+    double unpacked_lat;
+
     if (remote_iface_attr->addr_version == UCP_OBJECT_VERSION_V1) {
         /* Address v1 contains just latency overhead */
         return ucs_max(iface_attr->latency.c, remote_iface_attr->lat_ovh) +
                (iface_attr->latency.m * context->config.est_num_eps);
     } else {
-        return ucs_max(remote_iface_attr->lat_ovh,
-                       ucp_tl_iface_latency(context, &iface_attr->latency));
+        /* FP8 is a lossy compression method, so in order to create a symmetric calculation we pack/unpack the local latency as well */
+        packed_lat = UCS_FP8_PACK(LATENCY, ucp_tl_iface_latency(context, &iface_attr->latency) * UCS_NSEC_PER_SEC);
+        unpacked_lat = (UCS_FP8_UNPACK(LATENCY, packed_lat)) / UCS_NSEC_PER_SEC;
```

Comment:
- align by `=`
- pls drop the line to fit 80 symbols per line requirement
see clang code formatter suggestions - https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=45681&view=logs&jobId=a3e7afe8-b282-5869-712a-9857fbd4b4a9&j=cc064a77-22b5-56bf-ecc0-70b5fe764261&t=aedfd754-44a6-53e6-6843-8659d800fee2

Reply:
what about creating a helper function to do fp8 pack/unpack inside?

Reply:
Extra `()`?